src/main/java/edu/wpi/first/shuffleboard/MainWindowController.java

```diff
@@ -125,6 +128,16 @@ private void initialize() throws IOException {
     sourcesTab.setContent(networkTables.getView());
 
     widgetGallery.loadWidgets(Widgets.allWidgets());
+
+    root.addEventFilter(KeyEvent.KEY_PRESSED, e -> {
+      if (e.isShortcutDown() && e.getCode().isDigitKey()) {
+        // DIGIT1 is named "1", DIGIT2 is "2", etc.
+        int tabIndex = Integer.valueOf(e.getCode().getName()) - 1;
+        if (dashboard.getTabs().size() >= tabIndex) {
+          dashboard.getSelectionModel().select(tabIndex);
```

Comment:
this would probably be more correct to encapsulate inside of a void selectDashboard(int n) function, because there are presentational tabs we don't want to select (the adder tab)